web-app/src/app/tabs/battery/battery.component.css

```diff
@@ -1,65 +1,45 @@
 .header {
+	display: flex;
+	justify-content: space-between;
 	font-family: Arial, sans-serif;
 	font-size: 1.2rem;
-	background-color: grey;
-	color: rgba(255, 255, 255, 0.75);
-	padding-bottom: 1.2em;
+	background-color: transparent;
+	color: #D5FEFF;
+	padding-bottom: 0.2em;
 	padding-top: .2em;
-	border-radius: 25px;
-	margin-bottom: 25px;
-	margin-top: 10px;
+	margin-top: 0.1em;
+	margin-bottom: 0.1em;
+	width: auto;
 }
-.title-left {
-	float: left;
-	width: 50%;
-	margin-left: 5%;
+.aliveGroup {
+	margin-left: auto;
+	width: auto;
 }
-.title-right {
-	color: #73a2ef;
-	float: left;
-	width: 5%;
-	margin-left: 5px;
-
+.aliveGroup > .aliveText {
+	margin-left: 0.2em;
 }
-
 .circle {
 	float: left;
-	width: 15px;
-	height: 15px;
+	width: 1em;
+	height: 1em;
 }
 .circleAlive {
 	float: left;
-	width: 15px;
-	height: 15px;
+	width: 1em;
+	height: 1em;
```

Comment:
Just curious what the difference between circleAlive and circle is? They have the same css but one has margin-top 2px. The Battery page uses .circle and the Controls page uses the circleAlive. The circle shifts by 2px lower than the text on controls.

Reply:
I don't think it (intentionally) does anything different, I'll remove it and apply those changes to Battery